src/constants.js

```diff
@@ -32,8 +32,8 @@ export const MATCH_GIT_DIFF_FILE_SEPERATOR = /^diff --git /m;
 
 // Gerald Comment Regexes
 export const MATCH_REMOVEME_TAG_REGEX = /\#removeme/i;
-export const MATCH_GERALD_COMMENT_HEADER_REGEX = /^# Gerald:/;
-export const MATCH_COMMENT_HEADER_REGEX = /^### (Reviewers:|Required reviewers:|Notified:)$/m;
+export const MATCH_GERALD_COMMENT_HEADER_REGEX = /^# Gerald/;
+export const MATCH_COMMENT_HEADER_REGEX = /^<summary><b>(Reviewers|Required Reviewers|Notified)<\/b><\/summary>$/m;
```

Comment:
I wonder, are there any tests we can add to catch this more easily? Or some logging that can occur to highlight when this breaks?

Reply:
Yeah, good question... I'm unsure right now, but I do think we should be adding something!